doc/guide/Guide_Git_Development.ipynb

```diff
@@ -297,23 +278,27 @@
    "source": [
     "## Installing CLIMADA for development\n",
     "\n",
-    "0. **Install** [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)\n",
-    "   and [Anaconda](https://www.anaconda.com/) (or [Miniconda](https://conda.io/miniconda.html)).\n",
-    "\n",
-    "   Also consider installing Git flow. This is included with [Git for Windows](https://gitforwindows.org/)\n",
-    "   and has different implementations e.g. [here](https://skoch.github.io/Git-Workflow/) for\n",
-    "   Windows and Mac.\n",
+    "See [Installation](install.rst) for instructions on how to install CLIMADA for developers. You might need to install additional environments contained in ``climada_python/requirements`` when using specific functionalities. Also see {ref}`Apps for working with CLIMADA <app-overview>` for an overview of which tools are useful for CLIMADA developers. "
```

Comment:
This should link to an overview table in the installation guide. The link looks weird because it links to a subsection of an rst file from a jupyter notebook. If you build the documentation locally it should work. Did you try that?